cli/embedded-app/src/views/IframeEmbed.vue

```diff
@@ -71,10 +91,17 @@ export default {
     },
     updateFrameUrl(event) {
       this.frameUrl = event.detail;
+      this.showMenu = this.frameUrl === 'about:blank';
     }
   },
   mounted() {
     // Call the custom config set up on the CLI.
+    const oldSetupFrames = frameRouter.setupFrames;
+    frameRouter.setupFrames = (...args) => {
+      this.clientConfig = args[0];
+      oldSetupFrames.apply(frameRouter, args);
+    };
```

Comment:
The function proxying here is a bit clunky, but I don't want to expose the frame-router's internal client config publicly, and since this is part of the same repo/project as the frame router element itself, I think it's more reasonable than it would be otherwise.